spec/facades/foods_facade_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Facade' do
+   it 'get_food_search' do
+      search = FoodFacade.new
+      search_results = search.get_food_search("sweet potatoes")
+
+      expect(search_results).to be_a(Array)
```

Comment:
Could include assertions here about the contents of the array (i.e. all Food items, only 10 of them, etc)